src/main/scala/org/cs1331/gitsubmitter/GitSubmitterPlugin.scala

```diff
@@ -0,0 +1,30 @@
+package org.cs1331.gitsubmitter
+
+import java.time.LocalDateTime
+import sbt._
+import Keys._
+
+object GitSubmitterPlugin extends AutoPlugin {
+
+  override lazy val projectSettings = Seq(commands += submitCommand)
+
+  lazy val submitCommand = Command.command("submit") { (state: State) =>
+    val prismId = readLine("Enter your prism id: ")
```

Comment:
This is now unnecessary as my latest changes will append the username to the repo name if you pass `true` as the second parameter to `authenticateAndCreate`